src/Helpers.elm

```diff
@@ -12,6 +13,16 @@ type MaybeUpper
     | Lower String
 
 
+indent : Context -> Context
+indent c =
+    { c | indent = c.indent + 1 }
+
+
+deindent : Context -> Context
+deindent c =
+    { c | indent = c.indent - 1 }
+
+
```

Comment:
I think that's more suitable in ExContext than here.
Helpers.elm file is meant to be left agnostic against the rest of the project files